src/framework/operations.hpp

```diff
@@ -86,6 +86,9 @@ inline std::ostream& operator<<(std::ostream& stream, const OpType& type) {
   case OpType::save_statevec_ket:
     stream << "save_statevector_dict";
     break;
+  case OpType::save_mps:
+    stream << "save_mps";
```

Comment:
Should probably change this back to the long name
```suggestion
    stream << "save_matrix_product_state";
```

Reply:
done.